src/main/scala/no/ndla/audioapi/AudioApiProperties.scala

```diff
@@ -40,7 +40,7 @@ object AudioApiProperties extends LazyLogging {
   val SearchServer = propOrElse("SEARCH_SERVER", "http://search-audio-api.ndla-local")
   val SearchRegion = propOrElse("SEARCH_REGION", "eu-central-1")
   val RunWithSignedSearchRequests = propOrElse("RUN_WITH_SIGNED_SEARCH_REQUESTS", "true").toBoolean
-  val SearchIndex = "audios"
+  lazy val SearchIndex = propOrElse("SEARCH_ALIAS", "audios")
```

Comment:
Ah nå ser poenget ditt. Mindre dill for å oppnå det samme. Mer eksplisitthet,  mindre hoppe rundt for å skjønne hvor ting blir satt. 

Det bør i såfall gjøres likt i alle de 4 komponentene, det ble gjort med utangspunkt i hvordan det ble gjort i denne:
https://github.com/NDLANO/image-api/pull/26

og har dermed gjort det tilsvarende i alle de fire kompnentene sine search tester.